azure/table_azure_key_vault_secret.go

```diff
@@ -21,7 +21,7 @@ func tableAzureKeyVaultSecret(_ context.Context) *plugin.Table {
 		Get: &plugin.GetConfig{
 			KeyColumns:        plugin.AllColumns([]string{"vault_name", "name"}),
 			Hydrate:           getKeyVaultSecret,
-			ShouldIgnoreError: isNotFoundError([]string{"ResourceNotFound", "404"}),
+			ShouldIgnoreError: isNotFoundError([]string{"ResourceNotFound", "404", "403"}),
```

Comment:
What else can give us 403 errors from the `getKeyVaultSecret` function? If we have incorrect permissions, do we also get a 403 error?

Reply:
yes when wrong access policy set on vault in that time also get 403 error

Reply:
@anisadas Can you please try looking for the `Operation get is not allowed on a disabled secret.` error message instead of the 403 error code, as we don't want to ignore all permission errors for the get operation.